### ISSUE TITLE
Failing E2E: group-skipping

### ISSUE THREAD

**Issue by bgambocjaviniar** [SAP/spartacus#7019]
failing constantly since the refactoring on some parts of accessibility

more info: https://jkmaster.test.c3po.b2c.ydev.hybris.com/job/spartacus-regression-tests/909/consoleText

**Comment by bgambocjaviniar** [comment]
closes due to https://github.com/SAP/spartacus/issues/7028
